src/ai-bundle/src/AiBundle.php

```diff
@@ -494,8 +496,19 @@ private function processStoreConfig(string $type, array $stores, ContainerBuilde
             foreach ($stores as $name => $store) {
                 $arguments = [
                     new Reference($store['service']),
+                    new Definition(DistanceCalculator::class),
+                    $store['key'] ?? '_vectors',
```

Comment:
Shouldn't this key default come from the configuration or not been set if the default is already set via constructor?

Reply:
Hum, yes, by default, it's set to `_vectors` so the usage of the key is optional by design, I'll take a look to remove it.